aws_crt_common_ws/aws-crt-common-sys/build.rs

```diff
@@ -0,0 +1,337 @@
+        .add_file_to_build(Path::new("../../source/hash_table.c"))
+        .add_file_to_build(Path::new("../../source/lifo_cache.c"))
+        .add_file_to_build(Path::new("../../source/linked_list.c"))
+        .add_file_to_build(Path::new("../../source/linked_hash_table.c"))
+        .add_file_to_build(Path::new("../../source/log_channel.c"))
+        .add_file_to_build(Path::new("../../source/log_formatter.c"))
+        .add_file_to_build(Path::new("../../source/log_writer.c"))
+        .add_file_to_build(Path::new("../../source/logging.c"))
+        .add_file_to_build(Path::new("../../source/lru_cache.c"))
+        .add_file_to_build(Path::new("../../source/math.c"))
+        .add_file_to_build(Path::new("../../source/memtrace.c"))
+        .add_file_to_build(Path::new("../../source/priority_queue.c"))
+        .add_file_to_build(Path::new("../../source/process_common.c"))
+        .add_file_to_build(Path::new("../../source/ref_count.c"))
+        .add_file_to_build(Path::new("../../source/resource_name.c"))
+        .add_file_to_build(Path::new("../../source/ring_buffer.c"))
+        .add_file_to_build(Path::new("../../source/statistics.c"))
+        .add_file_to_build(Path::new("../../source/string.c"))
+        .add_file_to_build(Path::new("../../source/task_scheduler.c"))
+        .add_file_to_build(Path::new("../../source/thread_scheduler.c"))
+        .add_file_to_build(Path::new("../../source/thread_shared.c"))
+        .add_file_to_build(Path::new("../../source/uuid.c"))
+        .add_file_to_build(Path::new("../../source/xml_parser.c"));
+
+    #[cfg(windows)]
+    {
+        build_info
+            .add_file_to_build(Path::new("../../source/windows/clock.c"))
+            .add_file_to_build(Path::new("../../source/windows/condition_variable.c"))
+            .add_file_to_build(Path::new("../../source/windows/device_random.c"))
+            .add_file_to_build(Path::new("../../source/windows/environment.c"))
+            .add_file_to_build(Path::new("../../source/windows/file.c"))
+            .add_file_to_build(Path::new("../../source/windows/mutex.c"))
+            .add_file_to_build(Path::new("../../source/windows/process.c"))
+            .add_file_to_build(Path::new("../../source/windows/rw_lock.c"))
+            .add_file_to_build(Path::new("../../source/windows/system_info.c"))
+            .add_file_to_build(Path::new("../../source/windows/thread.c"))
+            .add_file_to_build(Path::new("../../source/windows/time.c"));
+
+        #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+        {
+            if build_info.get_toolchain().get_compiler().is_like_msvc() {
+                build_info.add_file_to_build(Path::new("../../source/arch/intel/msvc/cpuid.c"));
+            }
+        }
+
+        build_info
+            .add_link_target("Kernel32")
+            .add_link_target("BCrypt")
+            .add_link_target("Ws2_32");
+    }
+
+    #[cfg(not(Windows))]
```

Comment:
Is it not case-sensitive? I would not expect this to match on windows.

Reply:
good catch!